hawkbit-custom-theme-example/src/main/java/org/eclipse/hawkbit/app/MyUI.java

```diff
@@ -49,8 +50,8 @@ public class MyUI extends AbstractHawkbitUI {
     MyUI(final EventPushStrategy pushStrategy, final UIEventBus eventBus, final UIEventProvider eventProvider, final SpringViewProvider viewProvider,
             final ApplicationContext context, final DashboardMenu dashboardMenu, final ErrorView errorview,
             final NotificationUnreadButton notificationUnreadButton, final UiProperties uiProperties,
-            final VaadinMessageSource i18n) {
+            final VaadinMessageSource i18n, final ErrorHandler errorHandler) {
```

Comment:
please reference and close this PR: https://github.com/eclipse/hawkbit-examples/pull/54/files